devdoc/constbuffer_requirements.md

```diff
@@ -354,4 +388,62 @@ CONSTBUFFER_FROM_BUFFER_RESULT CONSTBUFFER_from_buffer(const unsigned char* sour
 
 **SRS_CONSTBUFFER_02_072: [** `CONSTBUFFER_from_buffer` shall succeed, write in `consumed` the total number of consumed bytes from `source`, write in `destination` the constructed `CONSTBUFFER_HANDLE` and return `CONSTBUFFER_FROM_BUFFER_RESULT_OK`. **]**
 
+**SRS_CONSTBUFFER_51_028: [** Shall set the state of `destination` to `CONSTBUFFER_WRITIABLE_STATE_SEAL`. **]**
+
 **SRS_CONSTBUFFER_02_073: [** If there are any failures then shall fail and return `CONSTBUFFER_FROM_BUFFER_RESULT_ERROR`. **]**
+
+
+### CONSTBUFFER_create_writiable_handle
+
+```c
+ CONSTBUFFER_HANDLE CONSTBUFFER_create_writable_handle (uint32_t size)
+```
+
+`CONSTBUFFER_create_writiable_handle` construct a new `CONSTBUFFER_HANDLE` of size `size` and state `CONSTBUFFER_WRITABLE_STATE_OPEN`.`CONSTBUFFER_HANDLE`supports two state `CONSTBUFFER_WRITABLE_STATE_SEAL` and `CONSTBUFFER_WRITABLE_STATE_OPEN`.If the state is `CONSTBUFFER_WRITABLE_STATE_OPEN` then buffers of `CONSTBUFFER_HANDLE` can be overridden (the same buffer can be reused to fill).
```

Comment:
>.If [](http://example.com/codeflow?start=244&length=3)

Nitpick: space after the dot. Allows for somewhat better formatting.

Reply:
Fixed.